File: 2-Patron/python/GestorDeUltimate.py
import Ultimate
class GestorDeUltimate():
    def __init__(self, ultimate):
        self.__ultimate = ultimate

    def subirDeNivel(self, incremento):
        return self.__ultimate.incNivel(incremento)
        
    def generarUltimate():
        return Ultimate("Basic", 10, 1)

File: 2-Patron/php/GestorDeUltimate.php
<?php
class GestorDeUltimate
{
  private $ultimate;

  function __construct($ultimate)
  {
    $this->ultimate = $ultimate;
  }

  function subirDeNivel($incremento)
  {
    return $this->ultimate->incNivel($incremento);
  }

  function generarUltimate()
  {
    return new Ultimate("Basic", 10, 1);
  }
}
?>
